src/ontology/edits/oeo-physical.omn

```diff
@@ -3545,17 +3548,22 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/411",
 Class: OEO_00010019
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "Synthetic liquid fuel is synthetic fuel produced in a power-to-liquid system from water and carbon dioxide using electric energy applying.",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "PtL fuel is a liquid synthetic fuel produced in a power-to-liquid system from water and carbon dioxide using electric energy applying.",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "P2L fuel",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "liquid synthetic fuel",
```

Comment:
Should we really use _liquid synthetic fuel_ as 'alternative term'?
I do not think that it would improve the handling with the oeo if _liquid synthetic fuel_ is obviously already the parent class.

Reply:
This is intended here as the term `liquid synthetic fuel` is in the domain very often used as synonym for what we label as `PtL fuel`. Using the alternative term shows here this ambiguity.

Reply:
Alright, I thought `synthetic liquid fuel` is only used in the domain because the former term had no 'alternative term'.